app/DoctrineMigrations/Version20180424190516.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20180424190516 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE team ADD active TINYINT(1) NOT NULL');
```

Comment:
Hva skjer når denne migreringen blir kjørt nå, blir alle team satt til inactive?

Burde alle eksisterende team bli satt til active?
`$this->addSql('UPDATE team SET active=1 WHERE active=0');`